### PR TITLE
Feature contact

### PR DESCRIPTION
**Pull request by wadeking98**
Created contact component.

_Note: the code assumes that a mock api server is running on localhost:3000._

You will need to start `json-server` on the same machine using the command `json-server --watch db.json`. `json-server` can be install with npm.

Here is an example `db.json` file:
```
{
  "reasonItems":[
      {"text":"My organization's information is incorrect", "value":"incorrect"},
      {"text":"I want my organization listed on OrgBook BC", "value":"listed"},
      {"text":"I want to become an OrgBook BC issuer", "value":"issuer"},
      {"text":"I want more information about APIs", "value":"apiInfo"},
      {"text":"other", "value":"other"}
    ]
}
```

### CODE REVIEW

File: src/services/api/v2/contact.service.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export default class Contact extends ApiResource {
+  baseVersion = "v2";
+  basePath = "feedback";
+
+  async postFeedback(feedback: ContactRequest|IncorrectInfoContactRequest): Promise<HttpResponse<ContactRequest|IncorrectInfoContactRequest>> {
+    return await Http.post<ContactRequest|IncorrectInfoContactRequest>(
+      this.formatEndpointUrl(""),
+      feedback,
+      {"Content-Type": "application/x-www-form-urlencoded"}
```
Comment (amanji) [inline src/services/api/v2/contact.service.ts:13]: Headers can be passed in via an Axios config. See change [here](https://github.com/petridishdev/orgbook-bc-client-vue/commit/1d8cbf87599b7b5b998a1fcaf2c013513cbde890#diff-78015307698270c08ff7c7830170acd5847318269e924388169144384195d84d) to Http service

File: src/store/modules/contact.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    requestTypes: [],
+  };
+  
+  const getters = {
+    requestTypes: (state: State): Array<string> => state.requestTypes,
+  };
+  
+  const actions = {
+      async fetchRequestTypes({ commit }: ActionContext<State, RootState>, url:string): Promise<void> {
+        await Http.get<HttpResponse<string>>(
```
Comment (amanji) [inline src/store/modules/contact.ts:41]: You likely won't be fetching this data from the API. Can import it directly from a client side enum

**Review by amanji — CHANGES_REQUESTED** [review]

File: src/services/api/v2/contact.service.ts (hunk 1 above)
Comment (wadeking98) [reply src/services/api/v2/contact.service.ts]: passed headers via axios config

File: src/store/modules/contact.ts (hunk 2 above)
Comment (wadeking98) [reply src/store/modules/contact.ts]: changed to get data from enum

**Review by amanji — APPROVED** [review]

**Pull request merged**